packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.1.use.md

```diff
@@ -0,0 +1,31 @@
+## Guidelines
+
+(See also: **Guidelines for All Form Controls** - _coming soon!_)
+
+### General
+
+-   Whereas checkboxes can be used to toggle options on and off, using a [toggle switch](#/components/FlatSelect) or [radio buttons](#/components/Radio) is preferable.
+-   A checkbox set should always have a title.
+-   Always put the most common options at the top of the checkbox list.
+-   If more than 7 options (checkboxes) are needed, it is preferable to group checkboxes logically under more than one set of options such that the set label represents the grouping logic.
+
+### Label
+
+-   Try to limit labels to one short line.
+-   Add instructions that indicate how many items can be selected, as some users might not know the difference between checkboxes and radio buttons.
+-   Consider listing options vertically, as horizontal listings can make it difficult to tell which label is related to which checkbox.
+-   [Partially selected states](#/components/Checkbox) should only be used when sub-options can be selected as children of a checkbox. In that case, the parent checkbox toggles all sub-options when clicked. In that case this state shows that only some sub-options are selected, not all of them.
```

Comment:
@Charles-ErickBG I don't get the last sentence here. The previous sentence says that all checkboxes are selected, so it seems like a bit of a contradiction. What am I not getting?

Reply:
I'm not sure I get what you mean.

The last sentence refers to one of the states that a checkbox can have, in a particular case, that is the PARTIALLY SELECTED state. I'll try to show you an example of when that state should be used.

  [  -  ] **Parent checkbox that allows to select and unselect all children (partially selected)**
  [ ✔] Option 1 selected
  [      ] Option 2 not selected
  [ ✔] Option 3 selected
  [      ] Option 4 not selected

Is it clearer to you?